src/Model/Locale.php

```diff
@@ -250,6 +273,14 @@ public function getCMSFields()
                         'Note: Per-domain specific locale can be assigned on the Locales tab'
                         . ' and will override this value for specific domains.'
                     )),
+                CheckboxField::create(
+                    'UseDefaultCode',
+                    _t(__CLASS__ . '.USE_X_DEFAULT', 'Use {code} as hreflang', ['code' => self::X_DEFAULT])
```

Comment:
Is there maybe a way we can avoid using `hreflang` here? It might be confusing for users

Reply:
Ah, possibly, except that's literally all it's used for. :P I can look at other description maybe. Like `SEO Language code`?

Reply:
Maybe both e.g. `hreflang (SEO language code)` ?

Reply:
Changed.